velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -121,6 +121,18 @@ struct YearFunction : public InitSessionTimezone<T> {
   }
 };
 
+template <typename T>
+struct QuarterFunction : public InitSessionTimezone<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE bool call(
```

Comment:
Please can you add an implementation for Date also. I recently updated all the extraction functions in this file for dates also ref https://github.com/facebookincubator/velox/pull/627

Reply:
Thank you for the suggestion! I'll add the implementation for Date.